src/SqlAsyncCollector.cs

```diff
@@ -171,6 +172,16 @@ private async Task UpsertRowsAsync(IEnumerable<T> rows, SqlAttribute attribute,
                 ObjectCache cachedTables = MemoryCache.Default;
                 var tableInfo = cachedTables[cacheKey] as TableInformation;
 
+                int timeout = 10;
+                string timeoutEnvVar = Environment.GetEnvironmentVariable("AZ_FUNC_TABLE_INFO_CACHE_TIMEOUT_MIN");
+                this._logger.LogDebugWithThreadId("TIMEOUT STRING HERE " + timeoutEnvVar);
```

Comment:
I'd remove this and then add a log message along the lines of "Overriding default table info cache timeout with new value {0}`

Reply:
Oops that was just for my debugging but yes I will update it.